COMPATIBILITY.md

```diff
@@ -3,7 +3,8 @@ The following table shows the compatibility of jaeger operator with different co
 
 | Jaeger Operator | Kubernetes           | Strimzi Operator   | Cert-Manager |
 |-----------------|----------------------|--------------------|--------------|
-| v1.33           | v1.19, v1.20, v1.21  | v0.23              | v1.6.1       |
+| v1.34.x         | v1.19 to v1.23       | v0.23              | v1.6.1       |
```

Comment:
We are testing Kubernetes 1.24 too. Should we place it here?

Reply:
good catch. yes :)